src/analysis/complex/isometry.lean

```diff
@@ -0,0 +1,185 @@
+/-
+Copyright (c) 2021 François Sunatori. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ruben Van de Velde, Yakov Pechersky, Heather Macbeth, Kevin Buzzard, Greg Price,
+François Sunatori
```

Comment:
You don't need to put everyone as an author here, putting your name is enough (it is not like, credits, but rather: who should we bug if something breaks in this file later on).